docs/src/tutorials/standalone/Canopy/simple_canopy.jl

```diff
@@ -0,0 +1,115 @@
+# # Simple Canopy Model Tutorial
+# Now that we've run a simple soil model simulation in [Getting Started](@ref getting_started.md),
+# let's try doing the same with the canopy model.
+
+# This tutorial sets up our CanopyModel on a column domain.
+# The model prognoses canopy temperature and liquid water content,
+# and diagnoses many more variables.
+# Here you'll see that the setup for different models is very similar,
+# and we'll highlight the changes between them.
+
+# First import the Julia packages we'll need.
+import ClimaParams as CP
+using ClimaUtilities: TimeVaryingInput
+using ClimaLand
+using ClimaLand.Domains
+using ClimaLand.Canopy
+import ClimaLand.Simulations: LandSimulation, solve!
+import ClimaLand.Parameters as LP
+using Dates
+import ClimaDiagnostics
+using CairoMakie, ClimaAnalysis, GeoMakie, Poppler_jll, Printf, StatsBase
+import ClimaLand.LandSimVis as LandSimVis
+
+# Choose a floating point precision, and get the parameter set,
+# which holds constants used across CliMA models.
+FT = Float32
+earth_param_set = LP.LandParameters(FT);
+
+# We will run this simulation on a point domain at a random lat/lon location.
+# This is different from the soil example, which ran on a column, because the
+# soil model has depth and the canopy model does not.
+longlat = FT.((34.1, -118.1))
+domain = Domains.Point(; z_sfc = FT(0.0), longlat);
+surface_space = domain.space.surface;
+
+# We choose the initial and final simulation times as DatesTimes, and a timestep in seconds.
+start_date = DateTime(2008);
+end_date = start_date + Second(60 * 60 * 72);
+dt = 1000.0;
+
+# Whereas the soil model takes in 2 forcing objects (atmosphere and radiation),
+# the canopy takes in 3 (atmosphere, radiation, and ground). Here we read in the
+# first two from ERA5 data, and specify prescribed conditions for the ground.
+# We also set up a constant leaf area index (LAI).
+# This differs from the soil example because we have the extra inputs of the ground conditions and LAI.
+era5_ncdata_path =
+    ClimaLand.Artifacts.era5_land_forcing_data2008_path(; lowres = true);
+atmos, radiation = ClimaLand.prescribed_forcing_era5(
+    era5_ncdata_path,
+    surface_space,
+    start_date,
+    earth_param_set,
+    FT,
+);
+ground = PrescribedGroundConditions{FT}();
```

Comment:
Just an idea for a future PR: I think there is some overlap between this and the "canopy_tutorial" but we can turn the canopy_tutorial into something more "advanced" - for example, we could force the canopy with the data from a fluxnet site

For this, we might note that this prescribes emissivity, albedo, temperature, and soil moisture

Reply:
I agree! It would be nice to make the canopy_tutorial.jl more complex now